Codeinwp/raft
# Adds secondary colors in theme.json + each palette

### Summary

Add the Secondary Accent color in the main theme.json file
and also added for each palette separately, the corresponding colors

### Will affect visual aspect of the product

NO

### Screenshots 


![image](https://github.com/Codeinwp/raft/assets/52494172/8c90e14e-ac2e-4aa3-a86b-e8348d1d4de3)



### Test instructions


- This changes are just in code. **We don't use the Secondary color anywhere, yet.**
- with this PR, the main palette should have a secondary color (in code). Same applies to all other color palettes.
- this can be checked by using the [Block Theme plugin](https://wordpress.org/plugins/create-block-theme/), which allows to see that each Color Palette and the Theme.json file have now a Secondary Accent Color

Closes #64  .
